apps/website/src/routes/dashboard/+page.svelte

```diff
@@ -155,28 +158,30 @@
           <Header tag="h3" color="secondary">Upcoming</Header>
         </svelte:fragment>
         <svelte:fragment slot="content">
-          <div class="flex flex-col gap-4">
-            {#each data.groupings.upcoming as { bills, household }}
-              <div class="card variant-outline-primary">
-                <Header tag="h4" class="card-header">
-                  {bills.billName} due on {bills.dueDate}
-                </Header>
-                <section class="p-4">
-                  {household.name}
-                </section>
-                <footer class="card-footer">
-                  <button
-                    class="btn btn-sm variant-filled"
-                    on:click={() =>
-                      console.info('somehow mark a payment to this bill')}
-                    >Pay bill</button
-                  >
-                </footer>
-              </div>
-            {:else}
-              No Upcoming bills
-            {/each}
-          </div>
+          <form action="/dashboard/payments?/payBill" method="post" use:enhance>
+            <div class="flex flex-col gap-4">
+              {#each data.groupings.upcoming as { bills, household }}
+                <div class="card variant-outline-primary">
+                  <Header tag="h4" class="card-header">
+                    {bills.billName} due on {bills.dueDate}
+                  </Header>
+                  <section class="p-4">
+                    {household.name}
+                  </section>
+                  <footer class="card-footer">
+                    <button
+                      class="btn btn-sm variant-filled"
+                      name="pay-bill-id"
+                      value={bills.id}
```

Comment:
Need to switch this over to using the payment drawer that we've got setup, but i think i can do that downstack.